src/pages/search/Search.tsx

```diff
@@ -37,74 +37,76 @@ const Search: React.FC<Props> = ({ match }) => {
       setSearchTerm(event.target.value);
     } else {
       setIsSearching(false);
-      setSearchTerm("")
+      setSearchTerm("");
     }
   };
 
   const renderSearchResults = () => {
     const results = searchContent(searchTerm, currentSegment);
 
-    return <>
-      <IonList>
-        {currentSegment === "constitution" && renderProvisions(results)}
-        {currentSegment === "rules" && renderRuleProvisions(results)}
-        {currentSegment === "guides" && renderTopics(results)}
-        {currentSegment === "cases" && renderCases(results)}
-      </IonList>
-      <IonContent class="ion-text-center ion-margin-top ion-padding-top">
-        {results.length <= 0 && (
-          <p>Nothing matches your search.</p>
-        )}
-      </IonContent>
-    </>
+    return (
+      <>
+        <IonList className="ion-no-padding">
+          {currentSegment === "constitution" && renderProvisions(results)}
+          {currentSegment === "rules" && renderRuleProvisions(results)}
+          {currentSegment === "guides" && renderTopics(results)}
+          {currentSegment === "cases" && renderCases(results)}
+        </IonList>
+        <IonContent class="ion-text-center ion-margin-top ion-padding-top">
```

Comment:
This doesn't seem to work well nested inside another ion-content. Maybe just use a p tag, rather than ion-content?

![image](https://user-images.githubusercontent.com/4178542/112955752-0b647700-9140-11eb-9bf6-78a4fd2e1dac.png)